src/components/ProjectSidebar.tsx

```
@@ -61,6 +66,29 @@ function DroppableProjectItem({
       } ${isOver && isDraggingTask ? "ring-2 ring-blue-500 bg-blue-500/10" : ""}`}
     >
+      {/* Reorder buttons - only show when not dragging a task */}
+      {!isDraggingTask && (
+        <div className="flex flex-col -my-1">
+          <button
+            onClick={(e) => { e.stopPropagation(); onMoveUp(); }}
+            disabled={isFirst}
+            className={`p-0.5 ${isFirst ? "text-zinc-700 cursor-not-allowed" : "text-zinc-600 hover:text-zinc-400"}`}
+            title="Move up"
+            aria-label={`Move ${project.name} up`}
+          >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)